fix: remove first person we from step 23 authors workshop (#67408)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-fcc-authors-page/641daae5e18eae4b562633e4.md

# --description--

> Finally, what if there's an error and the author data fail to load? An error should be shown in the UI. That's exactly what the `.catch()` method is for – handling errors.

Reviewer: ```suggestion
Finally, what if there's an error and the author data fail to load? Then you need to show an error in the UI. That's exactly what the `.catch()` method is for – handling errors.
```
